dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedAttributeTests.java

```diff
@@ -895,10 +896,90 @@ public void testHttpSessionOverridesHttpChannelConfig() throws Exception {
                     LOG.info("\n" + headerValue);
                      
                     if(headerValue.contains("Set-Cookie: JSESSIONID=")){
-                        // Assert Lax IS contained in the Cookie
+                        // Assert SameSite=None IS contained in the Cookie
                         assertTrue("The response did not contain the expected cookies header for the session", headerValue.contains("SameSite=None"));
                          // Assert Partitioned is NOT contained in the Cookie
                         assertTrue("The response did not contain the expected cookies header for the session", !headerValue.contains("Partitioned"));
+                    } else if (headerValue.contains("Set-Cookie: AddCookieName=")) {
+                        // Assert SameSite=None IS contained in the Cookie
+                        assertTrue("The response did not contain the expected cookies header for the session", headerValue.contains("SameSite=None"));
+                         // Assert Partitioned IS contained in the Cookie
+                        assertTrue("The response did not contain the expected cookies header for the session", headerValue.contains("Partitioned"));
+                    } else {
+                        fail("Unknown cookie encountered: " + headerValue);
+                    }
+                }
+            }
+        } finally {  
+            server.setMarkToEndOfLog();
+            server.restoreServerConfiguration();
+            server.waitForConfigUpdateInLogUsingMark(Collections.singleton(APP_NAME), false, "CWWKT0016I:.*PartitionedTest.*");
+        }
+    }
+
+     /**
+     * Verify HttpSession's Config overrides the HttpChannel Config. 
+     * All cookies except the Session Cookie is Partitioned. 
+     * This differs from the previous test becuase "cookieSameSite" is not set. 
+     * 
+     * Configuration Tested:
+     *    <httpSession cookiePartitioned="false" /> 
```

Comment:
Can we add this same test only without specifying anything in httpSession? Just to ensure that the default without explicitly setting cookiePartioned to false behaves the same?

Reply:
Should samesite still be set? 

I have a default session test here: https://github.com/OpenLiberty/open-liberty/blob/c0338c044f7b039c6ce56f6fb95dcd2fe471a637/dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedAttributeTests.java#L614